lib/client.js

```diff
@@ -233,6 +244,22 @@ function initializatorClient(requests, opts) {
       return driverResp.value
     }
 
+    async saveScreenshot(name, params = {}) {
+      if (!name) {
+        throw new Error(`Name is obligatory to save screenshot`);
+      }
+      const {
+          path = `${process.cwd()}/screenshots`,
+          encoding = 'base64',
+          format = 'png'
+        } = params,
```

Comment:
add const for every variable 
const screenshotPath = `${path}/${name}.${format}`
const  screenshot = await this.takeScreenshot()